.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Public
+public
```

Comment:
Because `public/bundle.js` is already set to be tracked by git for our repo, just adding public here will not cause it to be ignored. We'll have to also remove it from the files git is tracking:
```console
git rm --cached public/bundle.js
```

Reply:
updated